formats/developers/ome-units.txt

```diff
@@ -0,0 +1,253 @@
+The OME System of Units
+=======================
+
+.. topic:: Overview
+
+   This sections explains the OME system of Units.
+
+Implementation
+^^^^^^^^^^^^^^
+
+The `ome.xsd` schema file defines new type enumerations: UnitsLength,
+UnitsTime, UnitsPressure, UnitsAngle, UnitsTemperature,
+UnitsElectricPotential, UnitsPower and UnitsFrequency. These types are used
+throughout the model. Each new unit attribute is in a pair with an existing
+attribute, e.g. PinholeSize and PinholeSizeUnit. The Xsd-Fu application then
```

Comment:
Is this factually correct?  xsd-fu is not related to the OMERO units (`ome.units`) classes at all, is it?  Also, from the POV of an end user, internal details about xsd-fu are not important for using the API.  Could it be omitted entirely here?